.github/workflows/wolfictl-lint.yaml

```diff
@@ -18,4 +18,10 @@ jobs:
       uses: docker://ghcr.io/wolfi-dev/wolfictl:latest
       with:
         entrypoint: wolfictl
-        args: lint
\ No newline at end of file
+        args: lint
+    - name: Advisory sync
+      id: advisory
+      uses: docker://ghcr.io/wolfi-dev/wolfictl:latest
+      with:
+        entrypoint: wolfictl
+        args: "advisory sync-secfixes --warn *.yaml"
```

Comment:
@rawlingsj I think we can just drop the file part — the command should default to looking at all files in the current directory

```suggestion
        args: "advisory sync-secfixes --warn"
```

Reply:
YAY it passed and was super fast!